src/sdl2/ttf/font.rs

```diff
@@ -289,9 +289,11 @@ pub fn internal_load_font<'ttf,P:AsRef<Path>>(path: P, ptsize: u16) -> Result<Fo
 }
 
 /// Internally used to load a font (for internal visibility).
-pub fn internal_load_font_from_ll<'ttf,'r>(raw: *const ffi::TTF_Font, rwops: Option<RWops<'r>>)
-        -> Font<'ttf,'r> {
-    Font { raw: raw, rwops: rwops, _marker: PhantomData }
+pub fn internal_load_font_from_ll<'ttf,'r, RWOPS>(raw: *const ffi::TTF_Font, rwops: RWOPS)
+        -> Font<'ttf,'r>
+where RWOPS: Into<Option<RWops<'r>>>
```

Comment:
While I don't midn the whitespace changes, I do mind the place of the bracket after a function. A bracket beginning a function must not be on a separate line on its own, it must be on the same line as the function declaration.